src/rvya25-profile.adoc

```diff
@@ -0,0 +1,466 @@
+
+RVY64I is the mandatory base ISA for RVYA25U64 and is little-endian.  As
+per the unprivileged architecture specification, the `ECALL`
+instruction causes a requested trap to the execution environment.
+
+NOTE: The `ECALL` requirement is as specified in RVA23U64.
+
+==== RVYA25U64 Mandatory Extensions
+
+The following mandatory extensions are also mandatory in RVA23U64.
+
+- *M* Integer multiplication and division.
+- *A* Atomic instructions.
+- *F* Single-precision floating-point instructions.
+- *D* Double-precision floating-point instructions.
+- *B* Bit-manipulation instructions.
+- *Zicsr*  CSR instructions.  These are implied by presence of F.
+- *Zicntr* Base counters and timers.
+- *Zihpm* Hardware performance counters.
+- *Ziccif* Main memory regions with both the cacheability and
+  coherence PMAs must support instruction fetch, and any instruction
+  fetches of naturally aligned power-of-2 sizes up to min(ILEN,XLEN)
+  (i.e., 32 bits for RVYA25) are atomic.
+- *Ziccrse* Main memory regions with both the cacheability and coherence PMAs must support RsrvEventual.
+- *Ziccamoa* Main memory regions with both the cacheability and coherence PMAs must support all atomics in A.
+- *Zicclsm* Misaligned loads and stores to main memory regions with both the
+  cacheability and coherence PMAs must be supported.
+- *Za64rs* Reservation sets are contiguous, naturally aligned, and a
+   maximum of 64 bytes.
+- *Zihintpause* Pause hint.
+- *Zic64b* Cache blocks must be 64 bytes in size, naturally aligned in the
+address space.
+- *Zicbom* Cache-block management instructions.
+- *Zicbop* Cache-block prefetch instructions.
+- *Zicboz* Cache-Block Zero Instructions.
+- *Zfhmin* Half-precision floating-point.
+- *Zkt* Data-independent execution latency.
+- *V* Vector extension.
+- *Zvfhmin* Vector minimal half-precision floating-point.
+- *Zvbb* Vector basic bit-manipulation instructions.
+- *Zvkt* Vector data-independent execution latency.
+- *Zihintntl* Non-temporal locality hints.
+- *Zicond* Integer conditional operations.
+- *Zimop* may-be-operations.
+- *Zcmop* Compressed may-be-operations.
+- *Zcb* Additional compressed instructions.
+- *Zfa* Additional floating-Point instructions.
+- *Zawrs* Wait-on-reservation-set instructions.
+
+NOTE: The combination of "C" and "D" is incompatible with RVY as Zcd is an incompatible extension.
+
+Therefore the following mandatory extension is included instead:
+
+- *Zca* Compressed encodings excluding floating point loads and stores, including capability loads and stores for RVY
+
+The following mandatory extension is RVY specific:
+
+- *Zys* Sentry definition
+
+The following mandatory extension is in addition to RVA23U64:
+
+- *Zabhlrsc* Byte and halfword load-reserved and store conditional
+
+The following development option from RVA23U645 is mandatory in RVYA25U64:
+
+- *Zabha* Byte and halfword atomic memory operations.
+
+The following extension is mandatory in RVA23 but is _incompatible_ with RVY:
```

Comment:
It's fine for integer mode code? Just mask the address and let DDC authorise that.